styles/config/vocabularies/Docs/accept.txt

```diff
@@ -47,6 +47,7 @@ Atlassian
 autogenerate
 autogenerated
 [Au]toscal[e|er|ing]
```

Comment:
```suggestion
[Aa]utoscale
[Aa]utoscaler
[Aa]utoscaling
```

Reply:
For some reason regex for the ends of words doesn't seem to be working so can do this for now